actionview/test/template/form_options_helper_test.rb

```diff
@@ -18,12 +23,7 @@ class FormOptionsHelperTest < ActionView::TestCase
   end
 
   def setup
-    @fake_timezones = %w(A B C D E).map do |id|
-      tz = stub(:name => id, :to_s => id)
-      ActiveSupport::TimeZone.stubs(:[]).with(id).returns(tz)
-      tz
-    end
-    ActiveSupport::TimeZone.stubs(:all).returns(@fake_timezones)
+    @fake_timezones = %w(A B C D E).each_with_index.map{|id,index| TimeZoneStruct.new(id)}
```

Comment:
I'll hold off looking at this file to see if my proposed change in #21507 can slim the diff.